Sources/MeiliSearch/Indexes.swift

```diff
@@ -101,15 +101,21 @@ public struct Indexes {
    completes the query request. It returns a `Result` object that contains `[Index]`
    value if the request was successful or `Error` if a failure occurred.
    */
-  public static func getAll(config: Config, _ completion: @escaping (Result<[Index], Swift.Error>) -> Void) {
-    Request(config).get(api: "/indexes") { result in
+  public static func getAll(config: Config, params: IndexesQuery? = nil, _ completion: @escaping (Result<Results<Index>, Swift.Error>) -> Void) {
```

Comment:
Are we keeping `Result<Index>` and not `IndexResults` (like `DocumentsResult`  until the `GET /indexes` route has specific response fields that differ from other routes?